openapiGenerator.go

```diff
@@ -103,11 +104,20 @@ type openapiGenerator struct {
 
 	// @solo.io customizations to define schemas for certain messages
 	customSchemasByMessageName map[string]openapi3.Schema
+
+	// If set to true, OpenAPI schema will include schema to emulate behavior of protobuf oneof fields
+	protoOneof bool
+
+	// If set to true, native OpenAPI integer scehmas will be used for integer types instead of Solo wrappers
```

Comment:
Can you explain this in non `solo` terms. What do you mean by our wrapper?

Reply:
done